dio/pubspec.yaml

```diff
@@ -24,6 +24,7 @@ dependencies:
   http_parser: ^4.0.0
   meta: ^1.5.0
   path: ^1.8.0
+  web: ^0.5.1
```

Comment:
What is the min SDK requirement to use the web package?

Reply:
Seems to be 3.1.0

Reply:
> Seems to be 3.1.0

Maybe 3.3.0?